app/.storybook/preview.js

```diff
@@ -16,7 +16,7 @@ const router = createRouter({
 	],
 });
 
-const i18n = createI18n();
+const i18n = createI18n({ legacy: false });
```

Comment:
This was required for the emoji picker to be rendered in Storybook. Without this setting the following error was shown:
<img width="937" alt="Screenshot 2022-12-13 at 19 03 18" src="https://user-images.githubusercontent.com/5363448/207410588-38177582-9a91-475c-8acf-433ee539b92f.png">

I assume this is somehow related to the fact that the component is async. Since this option "only" applies to Storybook, I thought it to be acceptable and haven't looked at it more closely.